overlay/common/frameworks/base/packages/SystemUI/res/values/config.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+/*
+** Copyright 2009, The Android Open Source Project
```

Comment:
use copyright as in this file:
https://github.com/RiseOS/vendor_riseos/blob/android-9.0/overlay/common/frameworks/base/core/res/res/values/config.xml